### PR TITLE
Update revoke-ome-encrypted-mail.md

### PR DESCRIPTION
**Pull request by samschan-ms**
add condition on revoking in OWA

### CODE REVIEW

**Comment by samschan-ms** [comment]
@KCCross  please review and update string

**Review by KCCross — CHANGES_REQUESTED** [review]
See inline

File: microsoft-365/compliance/revoke-ome-encrypted-mail.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Once an email has been revoked, the recipient receives an error when they access
 
 ## How to revoke an encrypted message that you sent
 
+[!IMPORTANT] You can only revoke a mail if it is sent to a single recipient, such as Gmail and Yahoo recipients, that gets a link-based experience.  You cannot revoke a mail sent to Office 365 and Microsoft account recipients (for example, outlook.com users).
```
Comment (KCCross) [inline microsoft-365/compliance/revoke-ome-encrypted-mail.md]: (remove Important)
You can't revoke a mail you send to a recipient that uses a work or school account or a Microsoft account. You can only revoke a mail you sent to a single recipient that uses a social account such as gmail.com or yahoo.com.

**Comment by KCCross** [comment]
See requested changes. Iterate if you want.

File: microsoft-365/compliance/revoke-ome-encrypted-mail.md (hunk 1 above)
Comment (KCCross) [reply microsoft-365/compliance/revoke-ome-encrypted-mail.md]: Better - 
You can't revoke a mail that you sent to a recipient that uses a work or school account from Office 365 or Microsoft 365 or a user that uses a Microsoft account, for example, an outlook.com account. (if admins can, link to that. i can't remember offhand - sorry, oof response)

You can revoke a mail that you sent to a single recipient that uses a social account such as gmail.com or yahoo.com. In other words, you can revoke an email sent to a single recipient that received the link-based experience.

**Review by KCCross — COMMENTED** [review]
Nice!

**Review by KCCross — APPROVED** [review]

**Pull request merged**